docs/docs/writing-stories.mdx

```diff
@@ -18,12 +19,12 @@ Some people call this [Software 2.0](https://medium.com/@karpathy/software-2-0-a
 :::note
 Rasa X provides a UI for interactive learning, and you can use any user conversation
 as a starting point. See
-[Talk to Your Bot](https://rasa.com/docs/rasa-x/user-guide/share-assistant/#talk-to-your-bot)
+[Talk to Your Bot](https://rasa.com/docs/rasa-x/share-assistant/#talk-to-your-bot)
```

Comment:
this link actually broken

Reply:
Good catch, find + replace failed me here 🙈